rich-default/Dockerfile

```diff
@@ -30,7 +30,7 @@ RUN apt-get update && \
     rm -rf /var/lib/apt/lists/*
 
 RUN apt-get update && \
-    apt install -y --no-install-recommends octave octave-control octave-image octave-io octave-optim octave-signal octave-statistics && \
+    apt install -y --no-install-recommends octave octave-* && \
```

Comment:
Actually, we should list them explicitly, not as a `*`.

Reply:
There are a lot of octave packages to list them all out explicitly, if I remember correctly. The Dockerfile was able to build using `octave-*`.

Reply:
Yes, it works, but it is better practice to be explicit so that the docker image is reproducible (as possible).